rust/template/Cargo.toml

```diff
@@ -32,15 +32,18 @@ path = "./ovsdb"
 [dependencies]
 abomonation = "0.7"
 cpuprofiler = {version = "0.0.3", optional = true}
-differential-dataflow = "0.9"
-fnv = "1.0.2"
+#differential-dataflow = "0.9"
+differential-dataflow = { git = "https://github.com/ryzhyk/differential-dataflow", branch="0.10.2" }
+fnv="1.0.2"
 lazy_static = "1.3"
 libc = "0.2"
 num-traits = "0.2"
 rustop = "1.0.2"
 serde = {version = "1.0", features = ["derive"]}
 time = "0.1"
-timely = "0.9"
+#timely = "0.9"
+#timely = { git = "https://github.com/TimelyDataflow/timely-dataflow" }
```

Comment:
Can we clean this up or add a TODO to know why we kept it? Same comment applies to similar constellations as well.

Reply:
This will all go away with the next DD release...